src/cmake/externalpackages.cmake

```diff
@@ -186,6 +186,10 @@ if (OpenColorIO_FOUND)
     endif ()
 else ()
     set (OpenColorIO_FOUND 0)
+else ()
+    checked_find_package (yaml-cpp)
```

Comment:
I thin we might be able to skip these if we use the latest OCIO available...

Reply:
Yes, I think this is not necessary for modern OCIO with good exported cmake configs.

Reply:
Gone